### ISSUE TITLE
Issue regarding converting the YAMNet tf model weights to pytorch

### ISSUE THREAD

**Issue by rsarka34** [w-hc/torch_audioset#11]
Hello, 
I am trying to convert the Yamnet tf weights to Pytorch format... However, I'm just getting this error. I am new to pytorch.

**----> 9 import params as tf_params
ModuleNotFoundError: No module named 'params'**

Thanks in advance

**Comment by ilyakam** [comment]
The TL;DR is that you don't really need it. If you want PyTorch-compatible YAMNet model weights, just use [these](https://github.com/w-hc/torch_audioset/releases/download/v0.1/yamnet.pth).

But, if you're keen on converting them yourself, here's what worked for me as of today running on Ubuntu with conda v23.11.0:

1. Create and activate a new Anaconda environment for Python 3.6.7:
    ```sh
    conda create --name yamnet python=3.6.7
    conda activate yamnet
    ```

1. Install the necessary dependencies:
    ```sh
    conda install pytorch==1.9.1 torchvision==0.10.1 torchaudio==0.9.1 -c pytorch
    
    pip install pyyaml tensorflow==1.*
    ```

1. Install `torch_audioset` into the same Conda environment:
    ```sh
    git clone git@github.com:w-hc/torch_audioset.git
    cd torch_audioset
    pip install --editable .
    ```

1. Using [GitZip](https://kinolien.github.io/gitzip/), download `yamnet` from [this location and exact commit](https://github.com/tensorflow/models/tree/bd488858d610e44df69da6f89277e9de8a03722c/research/audioset/yamnet) (The clue was in [this file](https://github.com/w-hc/torch_audioset/blob/e8852c53becef811784754a2de9c4617d8db2156/torch_audioset/data/tflow_input_processing/yamnet_utils/features.py#L1-L4))

1. Unzip its contents to some folder; e.g.:
    ```sh
    unzip yamnet.zip -d ~/code/yamnet
    cd ~/code/yamnet
    ```

1. Copy [`convert_yamnet.py`](https://github.com/w-hc/torch_audioset/blob/e8852c53becef811784754a2de9c4617d8db2156/tf_2_torch/convert_yamnet.py) to the same folder:
    ```sh
    curl -O https://raw.githubusercontent.com/w-hc/torch_audioset/e8852c53becef811784754a2de9c4617d8db2156/tf_2_torch/convert_yamnet.py
    ```

1. Download the TensorFlow model weights to the same folder:
    ```sh
    curl -O https://storage.googleapis.com/audioset/yamnet.h5
    ```

1. Run the conversion script:
    ```sh
    python convert_yamnet.py
    ```

If all went well, the script will print `conversion succeed` at the very end.
